.travis.yml

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+notifications:
+  email: false
+
+services:
+  - mysql
+  - redis
+
+language: python
+
+python:
+  - "2.7"
+
+install:
+  - travis_retry pip install -e .[docs]
+
+script:
+  - sphinx-build -qnNW docs docs/_build/html
+  - python setup.py test -a "tests --cov-config .coveragerc --cov kwalitee --cov-report term-missing"
```

Comment:
move `"tests --cov-config .coveragerc --cov kwalitee --cov-report term-missing"` correctly to `pytest.ini` (see invenio-tags/pytest.ini)